packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/__tests__/accumulator.spec.tsx

```diff
@@ -11,7 +13,8 @@ const mock_connect_props = {
 jest.mock('Stores/connect.js', () => ({
     __esModule: true,
     default: 'mockedDefaultExport',
-    connect: () => Component => props => Component({ ...props, ...mock_connect_props }),
+    connect: () => (Component: React.FC<TAccumulatorProps>) => (props: TAccumulatorProps) =>
+        Component({ ...props, ...mock_connect_props }),
```

Comment:
if you remove 'connect' from component, please use here StoreProvider from deriv/stores

Reply:
Answered above.